docs/index.rst

```diff
@@ -191,7 +191,7 @@ You can also use dataclasses (standard library and Pydantic),
             ...
 
         @patch(path="/{user_id:uuid}", dto=PartialUserDTO)
-        async def partial_update_user(self, user_id: UUID4, data: PartialUserDTO) -> User:
+        async def partial_update_user(self, user_id: UUID4, data: DTOData) -> User:
```

Comment:
Would be better if the import for this was also added? `from litestar.dto import DTOData`?

Reply:
-Also DTOData[User]?
Ah thanks, I see that is a mistake by me, I will fix.

-Would be better if the import for this was also added?
There is already a `from litestar.dto import DTOData` on line 176, does this work in this context?

Reply:
ahh ok, I must have missed that, thanks again